news/5569-jsonschema-menuinst

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Validate menuinst JSON documents using the included `$schema` value. (#5569)
+
+### Deprecations
+
+* Menuinst validation warnings will be raised as errors in 25.11. (#5569)
+
+### Docs
+
+* <news item>
+
+### Other
+
+* `menuinst` is no longer a dependency. (#5569)
```

Comment:
There are requirements text files and a recipe in the repo. We should remove menuist from those files if they are there.

Reply:
duh stupid me.